fabfile/extension_confs/extension_default.ini

```diff
@@ -0,0 +1,477 @@
+[main]
+postgres_versions: ['14.5', '15beta4']
+extensions: ['citus', 'hll', 'topn', 'tdigest', 'auto_explain', 'bloom', 'adminpack', 'amcheck', 'btree_gin', 'btree_gist', 'citext', 'cube',
+             'dblink', 'dict_int', 'dict_xsyn', 'earthdistance', 'file_fdw', 'fuzzystrmatch', 'hstore', 'intarray', 'isn', 'lo', 'ltree',
+             'pageinspect', 'passwordcheck', 'pgcrypto', 'pgstattuple', 'pg_surgery', 'pg_trgm', 'pg_visibility', 'postgres_fdw', 'seg', 'tablefunc',
+             'test_decoding', 'tsm_system_rows', 'tsm_system_time', 'unaccent', 'uuid_ossp']
+test_count: 36
+
+[citus]
+contrib: False
+preload: True
+create: True
+configure: True
+repo_url: https://github.com/citusdata/citus.git
+git_ref: release-11.1
+relative_test_path: ./src/test/regress
+conf_string: '''
+    citus.hide_citus_dependent_objects=true
+    citus.enable_unsupported_feature_messages=false
+    citus.enforce_object_restrictions_for_local_objects=false
+    '''
+post_create_hook: after_citus_create
+
+[hll]
+contrib: False
+preload: False
+create: True
+configure: False
+repo_url: https://github.com/citusdata/postgresql-hll.git
+git_ref: v2.16
+relative_test_path: .
+
+[topn]
+contrib: False
+preload: False
+create: True
+configure: False
+repo_url: https://github.com/citusdata/postgresql-topn.git
+git_ref: v2.5.0
+relative_test_path: .
+
+[tdigest]
+contrib: False
+preload: False
+create: True
+configure: False
+repo_url: https://github.com/tvondra/tdigest.git
+git_ref: v1.4.0
+relative_test_path: .
```

Comment:
contrib and preload are explained above. If we should execute `create extension <ext>;`, then we set create True. For some of the extensions we should not create it. e.g. auto_explain, because it does not generate any object